prow/cmd/deck/jobs.go

```diff
@@ -34,6 +34,9 @@ const (
 
 type Job struct {
 	Repo        string `json:"repo"`
+	Branch      string `json:"branch"`
```

Comment:
For batched merges, I'll need to expand this to being a map of branch names (pull/1234) to SHAs. This is fine for now.